### DISCUSSION TITLE
Audio Bitrate?

### DISCUSSION THREAD

**Discussion by ghost** [eritislami/evobot#1120] [Q&A]
What bitrate does the bot default to when playing a youtube video? Youtube defaults to 128kbps, but youtube music can max out at 256kbps - is there a way to force 256kbps playback?

**Comment by wolf-yuan-6115** [comment]
I think there is no method to force ytdl to fetch 256kbps audio from YouTube music.
By the way, bitrate is not important at all, if your bot audio is crappy, you might have a network issue, not bitrate.

**Reply by ghost** [reply]
The audio is fine, I was just wondering if there was any way to increase it to 256

**Reply by alsoGAMER** [reply]
> I think there is no method to force ytdl to fetch 256kbps audio from YouTube music.

If i recall correctly with python it's possibile.

**Reply by wolf-yuan-6115** [reply]
> > I think there is no method to force ytdl to fetch 256kbps audio from YouTube music.
> 
> If i recall correctly with python it's possibile.

Yes, but performance will be horrible

**Comment by alsoGAMER** [comment]
I'm interested too, will do some research about it.
